app/styles/app.scss

```diff
@@ -725,74 +725,79 @@ hr {
                 border-left-color: lighten($brand-primary, 10%);
             }
         }
-        & > b {
+        .subject-container {
+            background-color: $brand-primary;
             display: inline-block;
-            padding: 5px 12px ;
-            position: relative;
-            border-radius: 4px;
-            &%pseudo {
-                content: '';
-                display: block;
-                position: absolute;
-                width: 0;
-                height: 0;
-                left: 100%;
-                top: 50%;
-                margin-top: -5rem;
-                border: {
-                    style: solid;
-                    width: 5rem 0 5rem 3rem;
-                    color: transparent transparent transparent $brand-primary;
+            transition: background-color 0.1s linear;
+            margin-left: -10px;
+            padding-left: 10px;
+            & > b {
+                display: inline-block;
+                padding: 5px 15px ;
+                position: relative;
+                border-radius: 4px;
+                &%pseudo {
+                    content: '';
+                    position: absolute;
+                    width: 0;
+                    height: 0;
+                    left: 100%;
+                    top: 0;
+                    border: {
+                        style: solid;
+                        width: 15px 0 15px 12px;
+                        color: transparent transparent transparent $brand-primary;
+                    }
+                }
+                &:before {
+                    @extend %pseudo;
+                    margin-left: 1px;
+                    border-left-color: white;
+                    z-index: 1;
+                }
+                &:after {
+                    @extend %pseudo;
+                    transition: border-color 0.1s linear;
+                    z-index: 2;
                 }
-            }
-            &:before {
-                @extend %pseudo;
-                margin-left: 1px;
-                border-left-color: white;
-                z-index: 1;
-            }
-            &:after {
-                @extend %pseudo;
-                transition: border-color 0.1s linear;
-                z-index: 2;
-            }
-            &:first-child {
-                padding-left: 0;
             }
             &:last-of-type {
-                padding-right: 2rem;
-                &:before, &:after {
+                & > b::before, & > b::after {
                     display: none;
                 }
-            }
-            & + b {
-                margin-left: 2rem;
-            }
-        }
-        & > .fa {
-            color: white;
-            cursor: pointer;
-            transition: color 0.1s linear;
-            &:hover {
-                color: lighten(red, 40%);
-            }
-            &:active {
-                color: lighten(red, 20%);
+                & > b > .cancel-subject {
```

Comment:
I notice that many of these selectors are extremely specific, and could break if the page saw even minor design changes. Is there a way to simplify this?

If I'm reading this correctly, this says:

"Only apply this style if this element is a child of a node with id preprint-form-subjects, which contains one or more elements called subject container, where the last of those contains a bold tag, in which case if that bolded element contains an element with class cancel-subject, only if this is the last subject container should we apply this style"

Some of this hierarchy might be necessary, but I'd encourage limiting the levels of nesting if at all possible. (including dependence on specific markup tags like `b`) We want the class to be unique enough not to accidentally style everything, but not so specific that a small design change breaks it entirely.